src/scripts.js

```diff
@@ -126,7 +126,8 @@ function processUserData() {
     store.getKey('sleepData'),
     user.id,
   );
-  // setApiData('http://localhost:3001/api/v1/sleep');
+  const userWeeklySleepData = userSleepData.slice(-7);
```

Comment:
We should be getting our user data through the getWeekly() function

Reply:
Looks like we are doing this for activity data on scripts line 114 and 115 as well. if we could change those over to using getWeekly() and getDataByDate.

Additionally, we should probably change the name on getCurrentData to getMostRecentDate.